lib/get-player.js

```diff
@@ -0,0 +1,17 @@
+module.exports = function(opts) {
+  let player = 'video';
+
+  if (opts.domain === 'youtube.com') player = 'youtube';
+  if (opts.domain === 'soundcloud.com') player = 'audio';
+  if (isAudio(opts.src)) player = 'audio';
+
+  return player;
+}
+
+function isAudio(src) {
+  const isSoundcloud = new RegExp('^(https?:)?//api.soundcloud.com\/tracks\/[0-9]+\/stream')
+        .exec(src);
+  const isAudio = new RegExp('^(https?:)?//*.+(.mp3|.opus|.weba|.ogg|.wav|.flac)')
```

Comment:
Definitely should DRY up these recurring RegExps. Maybe put inside a 'supported-formats' lib or something, just so you don't have to remember to update multiple spots if you add / remove formats.

fwiw, the regexp literal is compact and pretty nice, except you'd have to escape the // :

```js
const isAudio = /^(https?:)?\/\/*.+(.mp3|.opus|.weba|.ogg|.wav|.flac)/
```